pykechain/extra_utils.py

```diff
@@ -290,10 +290,13 @@ def update_part_with_properties(part_instance, moved_instance, name=None):
         elif prop_instance.type == PropertyType.REFERENCES_VALUE:
             if prop_instance.value:
                 properties_id_dict[moved_prop_instance.id] = [ref_part.id for ref_part in prop_instance.value]
-        elif prop_instance.type in (PropertyType.SINGLE_SELECT_VALUE, PropertyType.MULTI_SELECT_VALUE):
+        elif prop_instance.type == PropertyType.SINGLE_SELECT_VALUE:
             if prop_instance.model().options:
                 if prop_instance.value in prop_instance.model().options:
                     properties_id_dict[moved_prop_instance.id] = prop_instance.value
+        elif prop_instance.type == PropertyType.MULTI_SELECT_VALUE:
+            if all(value in prop_instance.model().options for value in prop_instance.value):
```

Comment:
This will not work if value is `None`

Reply:
I am retracing the same mistakes you did, ain't I?